### ISSUE TITLE
Laravel Pint does not work properly in environment with DevContainer.

### ISSUE THREAD

**Issue by jun-takagi-sddgrp** [open-southeners/vscode-laravel-pint#66]
**Describe the bug**

When saving a file or  `. /vendor/bin/pint` command, the terminal displays that the file has been successfully formatted with Laravel Pint, but the formatting is not reflected until the actual file is reopened.

**Expected behavior**

I want the formatting to be reflected in real time on the IDE.

**Screenshots**

![image](https://github.com/user-attachments/assets/a5dbd303-50f4-4245-8f89-f0cd1798a286)
![image](https://github.com/user-attachments/assets/8f23fe68-fcea-43ec-bb11-cd1d7fb4dc57)

**Environment:**

 - Operating system: Windows11
 - IDE / version: VSCode 1.94.2
 - Extension version 1.2.1
 
**Additional context**

This is my devcontainer.json.
```json
{
  "name": "Laravel App",
  "dockerComposeFile": ["compose.yaml"],
  "service": "app",
  "workspaceFolder": "/var/www/laravel-app",
  "customizations": {
    "vscode": {
      "extensions": [
        "esbenp.prettier-vscode",
        "open-southeners.laravel-pint",
        "ecmel.vscode-html-css",
        "formulahendry.auto-complete-tag",
        "onecentlin.laravel-extension-pack",
        "xabikos.JavaScriptSnippets"
      ],
      "settings": {
        "editor.formatOnSave": true,
        "editor.defaultFormatter": "esbenp.prettier-vscode",
        "laravel-pint.enable": true,
        "bladeFormatter.format.enabled": true,
        "[php]": {
          "editor.defaultFormatter": "open-southeners.laravel-pint"
        },
        "[blade]": {
          "editor.defaultFormatter": "shufo.vscode-blade-formatter"
        }
      }
    }
  }
}

```

**Comment by 200-0K** [comment]
Same problem, did you find a solution?

**Comment by jun-takagi-sddgrp** [comment]
No, we have used php-cs-fixer as an alternative because we could not figure out the cause.

**Comment by d8vjork** [comment]
This is a common behaviour in Windows, we will take a look at this shortly

**Comment by d8vjork** [comment]
@CicerBro mind sharing the extension config or logs to check how this command is running? Also mind that this extension spawn new subprocesses using the OS default shell, so could be one of these involved

Usually is the antiviruses, if you've some installed just check to exclude your projects' paths from their realtime scans

**Comment by JoaoKbral** [comment]
Brother, just brute force it 👍, I'm not sure if this will work for you but good luck
Got annoyed and created the pint.bat file
Just write this
@ECHO OFF
setlocal DISABLEDELAYEDEXPANSION
SET BIN_TARGET=%~dp0/pint
SET COMPOSER_RUNTIME_BIN_DIR=%~dp0
php "%BIN_TARGET%" %*

**Comment by d8vjork** [comment]
This is most likely fixed on newer releases

Feel free to open a new issue in case is not
